docs/reference/module-api/module-overview.md

```diff
@@ -933,17 +930,17 @@ Task functions takes the object `argv` as an argument, which includes the argume
 > Input:
 >
 > ```
-> node app jump sideways --foo --bar=whee --super-cool=totally
+> node app my-module:runIt taskOption --foo --bar=baz --use-color=green
```

Comment:
Apostrophe tasks are kebab-case. Sorry for the nitpick.

Reply:
👍